install_config/syncing_groups_with_ldap.adoc

```diff
@@ -637,8 +635,8 @@ augmentedActiveDirectory:
     groupMembershipAttributes: [ testMemberOf ] <4>
```

Comment:
@Miciah We may want to change `testMemberOf` to a more appropriate attribute name. I believe `testMemberOf` was chosen to not conflict with `memberOf` on the test LDAP server we have set up, which calculates `memberOf` instead of storing it as an attribute.